qutebrowser/app.py

```diff
@@ -82,6 +83,9 @@ def run(args):
     standarddir.init(args)
 
     if args.version:
+        # we need to init sql to print the sql version
+        # we can use an in-memory database as we just want to query the version
+        sql.init('')
```

Comment:
I improved the `test_version` output in 9b5227b987433762e1693c57a98f48729e0d5279 - turns out we get this on Ubuntu Trusty:

```
Traceback (most recent call last):
  File "/usr/lib/python3.4/runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.4/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/florian/qutebrowser/qutebrowser/__main__.py", line 29, in <module>
    sys.exit(qutebrowser.qutebrowser.main())
  File "/home/florian/qutebrowser/qutebrowser/qutebrowser.py", line 178, in main
    return app.run(args)
  File "/home/florian/qutebrowser/qutebrowser/app.py", line 88, in run
    sql.init('')
  File "/home/florian/qutebrowser/qutebrowser/misc/sql.py", line 43, in init
    .format(db_path, database.lastError().text()))
qutebrowser.misc.sql.SqlException: Failed to open sqlite database at :
```

I vaguely remember reading somewhere that passing `''` for an in-memory database only started working in some version - can you try passing `':memory:'` instead ([sqlite docs](https://www.sqlite.org/inmemorydb.html))?

(as a reminder: Feel free to merge `master` or cherry-pick that commit to your branch if needed, but please don't rebase)